CONTRIBUTING.md

```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Questions or need help with setup? Feel free to open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
```

Comment:
I'd replace "Questions" with "Have questions or ..." 

Also, to promote concision, I'd remove the "Feel free to" and start the second sentence with "Open an issue here:"